apisix/schema_def.lua

```diff
@@ -746,6 +749,20 @@ _M.plugins = {
 }
 
 
+_M.plugin_config = {
+    type = "object",
+    properties = {
+        id = id_schema,
+        desc = {type = "string", maxLength = 256},
```

Comment:
all of the `desc` should be the same, declare it as a variable and just quote it

Reply:
Will do it in the next PR.